larray/core/array.py

```diff
@@ -5866,14 +5870,17 @@ def divnot0(self, other):
         >>> b
         sex  M  F
              0  1
-        >>> a / b
-        nat\sex    M    F
-             BE  nan  1.0
-             FO  inf  3.0
         >>> a.divnot0(b)
         nat\sex    M    F
              BE  0.0  1.0
              FO  0.0  3.0
+
+        Compare this to:
+
+        >>> a / b                                  # doctest: +SKIP
```

Comment:
Is it tested in our unittests ?

Reply:
> Hum... Travis tests raise the following error:
> 
> > ModuleNotFoundError: No module named 'pywintypes'

I know. I introduced Windows specific code without noticing. I'll fix this next week.

Reply:
No idea... I will check that next week.

Reply:
yes it is (in test_array.py/test_binary_ops)